synapse/lib/layer.py

```diff
@@ -3692,6 +3692,7 @@ async def _editTagSet(self, nid, form, edit, sode, meta):
                 self.layrslab.delete(INDX_TOMB + abrv, nid, db=self.indxdb)
 
         kvpairs = []
+        kvpairs = [(INDX_TAG + abrv, nid)]
```

Comment:
@Cisphyx is this right? I couldn't get the tag scan to work until I added this. I also had to add the subsequent delete on L3750 which I hope is also right.

Reply:
We could adjust the abrv mapping to include the prefix on both the key/value side if we need to but I'd like to avoid that if possible which is why I'd like to figure out if we can adjust the tag perms checks in some way. For now I think the way you can do it would be to scan the abrv index by the INDX_TAG prefix then scan the layer index for each tag to build your tag mapping.

Reply:
> scan the abrv index by the INDX_TAG prefix

how do I do this? Everything I try returns no results.

Reply:
I think I have it now.